controllers/v1/library/individualController.js

```diff
@@ -0,0 +1,255 @@
+                    message: error.message || httpStatusCode.internal_server_error.message,
+                    errorObject: error
+                });
+            }
+        })
+    }
+
+    /**
+    * @api {get} /assessment/api/v1/library/individual/details/:librarySolutionId Details of Individual Library.
+    * @apiVersion 1.0.0
+    * @apiName Details of Individual Library
+    * @apiGroup Individual Library
+    * @apiSampleRequest /assessment/api/v1/library/individual/details/5ed5ec4dd2afa80d0f616460
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    * {
+    "message": "Successfully fetched individual solution detail",
+    "status": 200,
+    "result": {
+        "creator": "",
+        "about": "DCPCR Assessment Framework 2018",
+        "questions": [
+            {
+                "ecmName": "Learning Walk",
+                "question": "Are there any unnamed bottles lying around in school that may be harmful ? "
+            }
+        ],
+        "linkTitle": "",
+        "linkUrl": ""
+    }
+    */
+
+      /**
+      * Details of individual library
+      * @method
+      * @name details
+      * @param {Object} req - All requested Data.
+      * @returns {JSON} returns creator,about and questions details.
+     */
+
+    async details(req) {
+        return new Promise(async (resolve, reject) => {
+            try {
+                
+                let observationSolution = 
+                await individualHelper.details( 
+                    req.params._id  
+                );
+
+                return resolve(observationSolution);
+            } catch (error) {
+                return reject({
+                    status: error.status || httpStatusCode.internal_server_error.status,
+                    message: error.message || httpStatusCode.internal_server_error.message,
+                    errorObject: error
+                });
+            }
+        })
+    }
+
+    /**
+    * @api {get} /assessment/api/v1/library/individual/metaForm/:solutionId Individual Solution Meta Form
+    * @apiVersion 1.0.0
+    * @apiName Individual Solution Meta Form
+    * @apiGroup Individual Library
+    * @apiHeader {String} X-authenticated-user-token Authenticity token
+    * @apiSampleRequest /assessment/api/v1/library/individual/metaForm/5ed5ec4dd2afa80d0f616460
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    * "result": [
+        {
+            "field": "name",
+            "label": "Title",
+            "value": "",
+            "visible": true,
+            "editable": true,
+            "validation": {
+                "required": true
+            },
+            "input": "text"
+        }
+    ]
+    */
+
+     /**
+   * Individual meta form.
+   * @method
+   * @name metaForm
+   * @param {Object} req -request Data.
+   * @returns {JSON} - Individual meta form.
+   */
+
+   async metaForm(req) {
```

Comment:
@aman-tunerlabs - As discussed metaForm API need not be here, can be in observations and assessment controllers itself